### PR TITLE
PLT-573 | Add removeSupport in bulk API

### PR DESCRIPTION
**Pull request by aarshi0301**
## Change description

> Developer Notes : https://www.notion.so/atlanhq/Add-append-and-remove-support-for-bulk-endpoint-0133b5b48d684a94aa350d70ba72c491

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by checkaayush — APPROVED** [review]
Mostly LGTM. Left a minor clarification comment. Let's get 1 more approval before merge.

**Review by n5nk — CHANGES_REQUESTED** [review]
Please check 
- if we need to document the change , since we have priorities set for append and remove , the way API behaves. 
- Error handling during remove , if relationship is not present.
- Also please squash all the commits to one commit

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityGraphDiscoveryV2.java
Comment (aarshi0301) [reply repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityGraphDiscoveryV2.java:391]: Yes, If both are true 
Then there are 2 cases:

   1. if assets in both appendRelationshipAttribute and removeRelationshipAttribute are different
      i. create relation for assets in appendRelationshipAttribute if not exits.
     ii.  remove relation for assets in removeRelationshipAttribute if exits.
    
    2. if assets in both appendRelationshipAttribute and removeRelationshipAttribute are same (ex asset A):
          i. create relation for assets in appendRelationshipAttribute if not exits. (relation with assetA gets created)
          ii. remove relation for assets in removeRelationshipAttribute if exits. (relation with assetA gets deleted now)

**Comment by aarshi0301** [comment]
> Please check
> 
> * if we need to document the change , since we have priorities set for append and remove , the way API behaves.
> * Error handling during remove , if relationship is not present.
> * Also please squash all the commits to one commit

Squashed commits

**Review by aarshi0301 — COMMENTED** [review]
Done

**Review by n5nk — APPROVED** [review]

**Pull request merged**
